packages/showcases/svelte/stories/components/structure/VtmnListItem/VtmnListItem.stories.svelte

```diff
@@ -19,6 +19,18 @@
       description: 'Disable list item.',
       defaultValue: false,
     },
+    href: {
+      description: 'Redirection link.',
+      control: { type: 'text' },
+    },
+    target: {
+      description: 'Target of the link.',
+      control: { type: 'text' },
+    },
+    rel: {
+      description: 'Rel of the link.',
+      control: { type: 'text' },
+    },
```

Comment:
Maybe we can an example of the case we need, which is with a link AND a button as we can see in your screen shot. 

I don't know if it's a good practice but it can avoid some dirty usage of the component.
If the component allow it, we should show the best way to do it

Reply:
Yes, we can do this easily with conditional controls -> https://storybook.js.org/docs/react/essentials/controls#conditional-controls

But in fact, this one doesn't work with svelte =/

Reply:
you can't use <slot> in storybook ?